.gitmodules

```diff
@@ -0,0 +1,5 @@
+[submodule "syslibs/opcua/open62541"]
+	path = syslibs/opcua/open62541
+	url = https://github.com/open62541/open62541
+	shallow = true
+	branch = 1.2
```

Comment:
Don't you want to use the `pack/1.2` branch of open62541, which does not require recursive submodules?

Reply:
The current version does not require the recursive submodules (`deps/mdnsd` and `deps/ua-nodeset`). Hence, we don't need to stick with the *pack* versions.

I noticed, that the combination of *branch* and *shallow* are not a good idea for submodules (c.f. [shallow submodules](https://www.reddit.com/r/git/comments/5b8s8w/git_shallow_submodules/)) and that the current commit referenced by the submodule doesn't even belong to the 1.2 branch. (It's a commit from the open62541 master - my bad). I will downgrade to latest stable commit from branch 1.3. As mentioned in the [release candidate for 1.3](https://github.com/open62541/open62541/releases/tag/v1.3-rc1) the api is frozen for 1.3. So we should be fine with any future version of 1.3 (so that we can add the find_package CMake option with version 1.3).

Although, the [current changes to the nodestore api](https://github.com/open62541/open62541/commit/6b8db940e5fb4699c7bcde777fc7b21234cc947b) seem promising for future optimizations of the transformation concept. (i will add that to the library readme for future generations 😃)